draft-ietf-wimse-s2s-protocol.md

```diff
@@ -101,19 +101,20 @@ one of these two alternatives for standardization, once we have understood their
 
 ## Deployment Architecture and Message Flow
 
-Regardless of the transport between the workloads, we assume the following logical architecture:
+Regardless of the transport between the workloads, we assume the following logical architecture
+(numbers refer to the sequence of steps listed below):
 
 ~~~ aasvg
 +------------+               +------------+
-|            |               |            |
-|            |               | Workload B |
-| Workload A |==============>|            |
-|            |               |   +--------+
-|            |               |   |  PEP   |
+|            |      (2)      |            |
+|            |==============>| Workload B |
```

Comment:
Should we make the arrow on 2) go in both direction to accumulate mTLS?

Reply:
After discussion, maybe add authn to step 2 and leave step 3 with authz only?

Reply:
Please see the new version. I'm afraid the diagram's diff is a bit messy.